### PR TITLE
chore: improve GitHub integration, add CI config

### PR DESCRIPTION
**Pull request by bednar**
This PR introduce support for CI integration (next PRs will enable code style checks) add templates for GitHub issues, ...

### CODE REVIEW

File: setup.py
Comment (karel-rehor) [inline setup.py:1]: I see Python 3.7 has been removed from classifiers.  However on line 50 `python_requires='>=3.7'`, so 3.7 will still be supported or should line 50 be `python_requires='>=3.8'`

**Review by karel-rehor — CHANGES_REQUESTED** [review]
@bednar Can you double check line 50 of setup.py?  While python 3.7 was removed from classifiers, it is still included there as part of the logic of `python_requires`.

Otherwise, looks great.

File: setup.py
Comment (bednar) [reply setup.py:1]: Fixed, I've add `python_requires='>=3.8`

**Review by karel-rehor — APPROVED** [review]
Looks good.

**Pull request merged**